raaghu-elements/src/rds-dropdown-list/rds-dropdown-list.tsx

```
@@ -69,25 +69,26 @@
   showHint?: boolean; // To show hint in dropdown
   hint?: string; // Hint text
   showSelectedOption?: boolean; // To show selected option
+  showSearch?: boolean; // To show search input
 }
 
 const RdsDropdownList = (props: RdsDropdownListProps) => {
   const [showIcon, setShowIcon] = useState(
-    props.showIcon || props.showIcon == undefined ? true : false
+    props.showIcon || props.showIcon === undefined ? true : false
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)